Rock.Messaging.RabbitMQ/RabbitConnectionBase.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using RabbitMQ.Client;
+
+namespace Rock.Messaging.RabbitMQ
+{
+    public abstract class RabbitConnectionBase : IDisposable
```

Comment:
This base class provides very little value. The codebase would be cleaner if each if its inheritors manged its own connection.